sale_automatic_workflow/views/sale_order_views.xml

```diff
@@ -19,14 +19,4 @@
             </xpath>
         </field>
     </record>
-    <record id="view_order_form_inherit_sale_stock" model="ir.ui.view">
-        <field name="name">sale.order.form.sale.stock</field>
-        <field name="model">sale.order</field>
-        <field name="inherit_id" ref="sale_stock.view_order_form_inherit_sale_stock" />
-        <field name="arch" type="xml">
-            <xpath expr="//group[@name='sale_shipping']" position="inside">
-                <field name="all_qty_delivered" />
```

Comment:
I'm not sure this field must be removed from the view as it's still in the model. But as service product can still be delivered through timesheets without a picking, I guess we still need it.

BTW: `_compute_all_qty_delivered` might need some fixing.

Reply:
It's because `sale_stock` isn't installed anymore, so that overriding does not makes sense anymore.